### PR TITLE
VACMS-3974: Updates What's new in the CMS Block to appear on main Help page but not Announcements

### PR DESCRIPTION
**Pull request by ndouglas**
## Description

See #3974. 

## Testing done
Tested a few CMS Help Pages, including the Announcements page.

Added a couple simple Behat tests to ensure that new CMS Help pages will display the block and that the Announcements page will not.

## Screenshots
Screen Shot 2021-01-11 at 4 19 43 PM
Screen Shot 2021-01-11 at 4 19 51 PM

## QA steps

As user _uid_ with _user_role_
1. Do this
1. Then that
1. Then validate Acceptance Criteria from issue
- [ ] This
- [ ] That
- [ ] The other thing

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [x] No announcement is needed for this code change. 
  - [x] Merge & carry on unburdened by announcements

### CODE REVIEW

**Review by oksana-c — DISMISSED** [review]
👍

File: config/sync/core.entity_form_display.taxonomy_term.health_care_service_taxonomy.default.yml [hunk 1]
```diff
@@ -94,7 +94,6 @@ content:
     region: content
     settings: {  }
     third_party_settings: {  }
-  revision_log_message:
```
Comment (acrollet) [inline config/sync/core.entity_form_display.taxonomy_term.health_care_service_taxonomy.default.yml:97]: @ndouglas this change needs to be reverted - I'm not sure if all taxonomies should be this way but pretty sure this one should.

c.f. https://github.com/department-of-veterans-affairs/va.gov-cms/pull/3951#pullrequestreview-565770381

Comment (acrollet) [reply config/sync/core.entity_form_display.taxonomy_term.health_care_service_taxonomy.default.yml:97]: @swirtSJW should all taxonomies have this config, or just this one + maybe some others?

Comment (ndouglas) [reply config/sync/core.entity_form_display.taxonomy_term.health_care_service_taxonomy.default.yml:97]: ugh, I meant to commit only the one file and apparently failed.  thanks

**Review by acrollet — APPROVED** [review]

**Pull request merged**